### PR TITLE
Portfolio 

### PR DESCRIPTION
**Pull request by jennymegan**
Second draft html and css.

### CODE REVIEW

File: Portfolio.html [hunk 1]
```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE = html>
```
Comment (mporam) [inline Portfolio.html]: no `=` in a doctype:
`<!DOCTYPE html>`

File: Portfolio.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Jenny Allton-Twist</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">
```
Comment (mporam) [inline Portfolio.html]: generally its best practise to avoid capital letters in file names

File: Portfolio.html (hunk 1 above)
Comment (mporam) [inline Portfolio.html]: you want to name this file index.html - ill explain later in the week

File: Portfolio.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
+		 </ul>
+		 <div>
```
Comment (mporam) [inline Portfolio.html]: is there a reason you have an empty div here?

File: Portfolio.html [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+			 
+	 <!-- <div><img src="" alt=""/> Place for logo -->
+		 </div>
+	</nav>
+
+
+
+	<div class="intropage" id="home">
+		<div class="container">
+			<h1>JENNY ALLTON-TWI<span clasmate s="spinner">S</span>T</h1>
```
Comment (mporam) [inline Portfolio.html]: clasmate? this is not valid html. I assume you meant `class="spinner"`

File: Portfolio.html [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+		</div>
+	</div>
+
+
+
+	<div class="portfolio" id="portfolio"><h2>Portfolio</h2>
```
Comment (mporam) [inline Portfolio.html]: drop the h2 onto the next line

File: Portfolio.html [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+		<div class="container">
+			<div class="box">
+				<a href="project.html"><img src="tech.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
```
Comment (mporam) [inline Portfolio.html]: the 0 and the h4 are the wrong way round. the project title is a heading, so a `h4`, the date is just additional info, so a `p`

File: Portfolio.html [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+				<h4>October 2020, Mayden Academy, Bath UK</h4>
+			</div>
+			<div class="box">
```
Comment (mporam) [inline Portfolio.html]: box is not a semantic class name

File: PortfolioStyle.css [hunk 8]
```diff
@@ -0,0 +1,379 @@
...
+	box-sizing: border-box;
+}  
+	
+/* Using rems - set up to be 10px = 1 rem*/
+
+body {
+	font-size: 62.5%;
+	font-family: "Hind Guntur";
+	font-weight: 300;
+	color: #000500 ;
```
Comment (mporam) [inline PortfolioStyle.css]: unnecessary space before the `;`

File: PortfolioStyle.css [hunk 9]
```diff
@@ -0,0 +1,379 @@
...
+	font-size: 1.2rem;
+	text-align: center;
+}
+
+
+/*Nav Bar*/
+
+nav {
+	overflow: hidden;
+	position:fixed;
```
Comment (mporam) [inline PortfolioStyle.css]: missing space after the `:` - try to keep everything consistant

File: PortfolioStyle.css [hunk 10]
```diff
@@ -0,0 +1,379 @@
...
+  	margin-right: auto;
+  	object-fit: cover;
+	width: 100%;
+  	height: 25vh;
+  	background-color: #CED3DC;
+}
+
+
+.medbox img {
+  	display: block;
```
Comment (mporam) [inline PortfolioStyle.css]: something funny happening here with your indentation

File: PortfolioStyle.css [hunk 11]
```diff
@@ -0,0 +1,379 @@
...
+
+.fa:hover {
+ 	 opacity: 0.7;
+}
+
+/* Set a specific color for each brand */
+
+.fa-twitter {
+	background: #55ACEE;
+	color: white;
```
Comment (mporam) [inline PortfolioStyle.css]: please use hex for all colors where possible, generally never use color word values

File: PortfolioStyle.css [hunk 12]
```diff
@@ -0,0 +1,379 @@
...
+		height: auto;
+	}
+	.container .box {
+		width: 95%;
+		margin: 10px 10px;
+	}
+	
+}
+
+/*@media screen and (max-height: 890px) {
```
Comment (mporam) [inline PortfolioStyle.css]: if this sint needed, please remove it

File: PortfolioStyle.css [hunk 13]
```diff
@@ -0,0 +1,379 @@
...
+
+@media screen and (max-width: 600px) {
+
+	
+	.intropage .container h1 {
+		font-size: 4rem;
+		padding: 10px;	
+	}
+	
+	nav li{
```
Comment (mporam) [inline PortfolioStyle.css]: missing a space before the `{`

File: PortfolioStyle.css [hunk 14]
```diff
@@ -0,0 +1,379 @@
...
+		font-size: 1rem;
+	}
+	
+	.project.container {
+		width: 100vw;
+	}
+
```
Comment (mporam) [inline PortfolioStyle.css]: alot of unnecessary line breaks in this area...

File: project.html [hunk 15]
```diff
@@ -0,0 +1,43 @@
...
+			<li><a href="portfolio.html">Home</a></li>
+			<li><a href="portfolio.html#portfolio">Portfolio</a></li>
+			<li><a href="portfolio.html#aboutme">About Me</a></li>
+			<li><a href="portfolio.html#contact">Contact</a></li>
+		 </ul>
+		 <!-- <div><img src="" alt=""/> Place for logo -->
+	</nav>
+
+	<div class="container project">
+		<div class="projectimg"><img src="projectimg.jpg" alt="Photos of Project" width="600"></div>
```
Comment (mporam) [inline project.html:24]: dont put widths on images in html, use CSS

**Review by mporam — CHANGES_REQUESTED** [review]

File: Portfolio.html (hunk 4 above)
Comment (norayna) [reply Portfolio.html]: typed in the editor instead of in the terminal by accident?

File: Portfolio.html (hunk 3 above)
Comment (jennymegan) [reply Portfolio.html]: Was going to add a little logo into the nav bar but hadn't yet located a non-cheesy one. Still thinking about it.

File: Portfolio.html (hunk 4 above)
Comment (jennymegan) [reply Portfolio.html]: Typo as Rayna pointed out - I must have been trying to open textmate and wondering why it wasn't typing inthe terminal... doh

File: project.html (hunk 15 above)
Comment (jennymegan) [reply project.html:24]: I'm going to redo this whole page, the layout doesn't work.

File: indexstyle.css [hunk 16]
```diff
@@ -13,33 +15,37 @@
 
 body {
 	font-size: 62.5%;
-	font-family: "Josefin Sans";
-	font-weight: 100;
-	background-color: #FCF7F8;
+	font-family: "Hind Guntur";
+	font-weight: 300;
+	color: #000500 ;
```
Comment (mporam) [inline indexstyle.css]: extra space before the `;`

File: indexstyle.css [hunk 17]
```diff
@@ -99,58 +106,85 @@ li a:hover {
...
 .container {
 	display: flex;
-	height: 100vh;
-	justify-content: space-between;
+	height: 90vh;
+	justify-content: space-evenly;
 	flex-wrap: wrap;
 	align-items: center;
-	padding: 20px;
+	padding: 20px 20px;
```
Comment (mporam) [inline indexstyle.css]: this does nothing? `padding: 20px;` is identical, u can just remove the second `20px`

**Review by mporam — CHANGES_REQUESTED** [review]

File: Portfolio.html (hunk 6 above)
Comment (jennymegan) [reply Portfolio.html]: I like the date and location as a bolded or more important thing, so I'm leaving as an h4 but I have put a heading h2 above the p now.

File: PortfolioStyle.css (hunk 8 above)
Comment (jennymegan) [reply PortfolioStyle.css]: pedant

File: PortfolioStyle.css (hunk 10 above)
Comment (jennymegan) [reply PortfolioStyle.css]: looks good in textmate?

**Review by mporam — CHANGES_REQUESTED** [review]
code looks good but the date/locations of the projects fall off the bottom of the boxes at certain screen sizes

**Review by mporam — CHANGES_REQUESTED** [review]
the name heading at the top is now broken

**Review by mporam — APPROVED** [review]

**Pull request merged**